### PR TITLE
[doc] netaddress: Make IPv4 loopback comment more descriptive

### PR DESCRIPTION
**Pull request by dongcarl**
This also makes the comment match the IPv6 comment just below this hunk.

### CODE REVIEW

File: src/netaddress.cpp [hunk 1]
```diff
@@ -182,7 +182,7 @@ bool CNetAddr::IsTor() const
 
 bool CNetAddr::IsLocal() const
 {
-    // IPv4 loopback
+   // IPv4 loopback (127.0.0.0/8 or 0.0.0.0/8)
```
Comment (laanwj) [inline src/netaddress.cpp]: Indentation should always be 4 spaces

Comment (MarcoFalke) [reply src/netaddress.cpp]: Don't forget to install `clang-format` and run the https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy script to preempt whitespace nitpicking.

**Comment by dongcarl** [comment]
Fixed!

**Comment by fanquake** [comment]
utACK 87aa0b4

The`error C2065: 'g_logger': undeclared identifier` Appveyor failure has been fixed in #15347.

**Review by kristapsk — APPROVED** [review]
utACK 6180b5f32bbcd7ef18db772290954161806e609e

**Pull request merged**